src/services/shifts.service.ts

```diff
@@ -68,22 +68,16 @@ export class ShiftsService extends Service {
       .then((res) => Promise.resolve(options?.rawResponse ? res : new Shift(res.data)));
   }
 
-  delete(id: number): Promise<number>;
-  delete(id: number, options: { rawResponse: true } & Options): Promise<AxiosResponse<any, any>>;
-  delete(id: number, options: Options): Promise<number>;
-  delete(id: number, options?: Options) {
-    return super
-      .fetch<ApiShift>({ url: `${this.apiPath}/${id}`, method: 'DELETE' })
-      .then((res) => Promise.resolve(options?.rawResponse ? res : res.status));
-  }
+  delete(data: number | number[]): Promise<number>;
+  delete(data: number | number[], options: { rawResponse: true } & Options): Promise<AxiosResponse<any, any>>;
+  delete(data: number | number[], options: Options): Promise<number>;
+  delete(data: number | number[], options?: Options) {
```

Comment:
Looking good and think this is simpler then a separate method.
I would probably keep the param name related to `id` though (perhaps `ids`?) - `data` is a bit vague for a consumer of this and might be unclear what should be provided